README.md

```diff
@@ -8,12 +8,289 @@ The executables found on the [release tab of this repository](https://github.com
 
 The script `interoperability_report.py` generates automatically
 the verification between two executables of these interoperability tests.
-The tests that the script runs must be defined previosly (for example
+The tests that the script runs must be defined previously (for example
 `test_suite.py`).
 Once the script finishes, it generates a report with the result
 of the interoperability tests between both executables.
 
-## Options of interoperability_report
+You can run the script either automatically or manually.
+## Table of contents
+
+* 1\. [Run the tests automatically](#automatically)
+
+    * 1.1. [Create a release](#release)
+
+      * 1.1.1. [When to create a release](#whenrelease)
+
+      * 1.1.2. [Release and tag name](#releasename)
+
+      * 1.1.3. [Process of creating the release](#createrelease)
+
+    * 1.2. [Process of uploading the executable](#executable)
+
+    * 1.3. [Process of generating the report](#generatereport)
+
+      * 1.3.1. [How to delete a report](#deletereport)
+
+      * 1.3.2. [Report](#report)
+
+    * 1.4. [Reporting failures](#failures)
+
+      * 1.4.1. [How to create a label](#createlabel)
+
+    * 1.5. [Where can I find the last report?](#lastreport)
+
+* 2\. [Run the tests manually](#manually)
+
+    * 2.1. [Options of interoperability_report](#options)
+
+      * 2.1.1. [Example of use interoperability_report](#example)
+
+
+    * 2.2. [Requirements](#requirements)
+
+    * 2.3. [Using virtual environments](#virtual)
+
+      * 2.3.1. [Create virtual environment](#createvirtual)
+
+      * 2.3.2. [Activate virtual environment](#activatevirtual)
+
+      * 2.3.3. [Install requirements](#installvirtual)
+
+* 3\. [How to make changes in the repository](#changes)
```

Comment:
We should reorganize the outline, it should show the correct order in which things will happen: create an executable, upload it, run test, etc...

I'm putting here some suggestions without links

```suggestion
1. [Requirements](#requirements)
    1.1. [Using virtual environments](#virtual)
    1.1.1. [Create virtual environment](#createvirtual)
    1.1.2. [Activate virtual environment](#activatevirtual)
    1.1.3. [Install requirements](#installvirtual)
2. Run Interoperability Test
    explain how to run interoperability test manually
 2.1. [Options of interoperability_report](#options)
 2.2. [Example of use interoperability_report](#example)
 2.3 Report
 explain some information about the report, that it is in JUnit and that is the only format supported so far.
3. Automation with GitHub actions
  explain how to run the interoperability test with GitHub actions
4. Workflow
 4.1 Create Executable
 4.2. Upload Executable
 4.3 Create a New Release
       here we should explain how to create a new release, including the tag information, the versioning, when the new releases are created, 
    NOTE: we don't need to explain how to delete a report, theoretically the reports won't be deleted
 4.3. Report Bugs
    say also that this should set the corresponding label, we don't need to say how to create a label, they should be created
5. Contribution
   here we should explain how to contribute, although this section won't be mandatory
```

Reply:
I still need to review the content of the file, but we should start by organizing it better